### PR TITLE
doc/cephadm: Add  creation method of OSD

### PR DESCRIPTION
**Pull request by yangxiaolaing**
ceph orch daemon add osd command  is does not supported for device partition,when you have not raw block device,you can create lvm through device partition,then you can use ceph orch daemon add osd <host>:<lvm-path> command create osd
Signed-off-by: yangxiaoliang <yangxiaoliang07@163.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by adk3798 — APPROVED** [review]
LGTM

File: doc/cephadm/services/osd.rst [hunk 1]
```diff
@@ -144,6 +144,18 @@ There are a few ways to create new OSDs:
 
     ceph orch daemon add osd host1:data_devices=/dev/sda,/dev/sdb,db_devices=/dev/sdc,osds_per_device=2
 
+* Create an OSD from a specific lvm on a specific host:
```
Comment (anthonyeleven) [inline doc/cephadm/services/osd.rst:147]: I don't typically see `lvm` used this way, I suggest "Create an OSD on a specific LVM logical volume on a specific host" instead.

File: doc/cephadm/services/osd.rst [hunk 2]
```diff
@@ -144,6 +144,18 @@ There are a few ways to create new OSDs:
...
+
+  .. prompt:: bash #
+
+    ceph orch daemon add osd *<host>*:*<lvm-path>*
+
+  For example:
+
+  .. prompt:: bash #
+
+    ceph orch daemon add osd host1:/dev/vg_osd/lvm_osd
```
Comment (anthonyeleven) [inline doc/cephadm/services/osd.rst:157]: suggest an example that is more reflective of a host having more than one OSD.  Something like

"ceph orch daemon add osd host1:/dev/vg_osd/lvm_osd1701"

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
Thanks for contributing! I always like to see specific examples added to documentation.  Sometimes it's not always clear how to translate descriptions into what one actually should type.

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Thanks.  You’ll need to amend the PR and commit comments with signed-off-by for tests to pass. 

https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change

**Pull request closed without merging**